src/main/java/io/boomerang/security/util/MultiReadHttpServletRequest.java

```diff
@@ -28,18 +27,16 @@
 import javax.servlet.http.Part;
 import org.springframework.util.StreamUtils;
 
-/*
- * Reference: https://www.baeldung.com/spring-reading-httpservletrequest-multiple-times
- */
```

Comment:
Let's leave this in. We need to know the reference for where this method is duplicated from. it helps retain knowledge.

Reply:
I will put this back.